tools/dev.server.js

```diff
@@ -17,7 +17,7 @@ compiler.hooks.done.tap('Verdaccio Dev Server', () => {
 });
 
 new WebpackDevServer(compiler, {
-  contentBase: `${env.DIST_PATH}`,
```

Comment:
Why this?

Reply:
we don't need to wrap it in a string literal